docker-compose.yaml

```diff
@@ -27,15 +27,20 @@ services:
   app:
     build:
       context: .
+      dockerfile: Dockerfile
     image: ghcr.io/ardoco/rest
     container_name: ardoco-rest-api
     restart: always
     ports:
-      - "127.0.0.1:8080:8080"
+      - "127.0.0.1:8081:8080"
     depends_on:
       - redis
     networks:
       - app-network
+    volumes:
+      - ~/.m2:/root/.m2
+#      - ../REST:/build
+#      - ../REST/target:/target
```

Comment:
Do we need these changes ? Probably also move to "docker-compose-template.yml" and add docker-compose.yml to .gitignore